content/zh/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -0,0 +1,451 @@
+
+例子：`kubernetes.io/hostname=ip-172-20-114-199.ec2.internal`
+
+用于：Node
+
+Kubelet 使用主机名填充此标签。请注意，可以通过将 `--hostname-override` 标志传递给 `kubelet` 来更改“实际”主机名。
+
+此标签也用作拓扑层次结构的一部分。 有关详细信息，请参阅 [topology.kubernetes.io/zone](#topologykubernetesiozone)。
+
+### kubernetes.io/change-cause {#change-cause}
+
+例子：`kubernetes.io/change-cause=kubectl edit --record deployment foo`
+
+用于：所有对象
+
+此注解是对更改某些内容的最佳猜测。
+
+将 `--record` 添加到可能会更改对象的 `kubectl` 命令时会填充它。
+
+### kubernetes.io/description {#description}
+
+例子：`kubernetes.io/description: "Description of K8s object."`
+
+用于：所有对象
+
+此注解用于描述给定对象的特定行为。
+
+### kubernetes.io/enforce-mountable-secrets {#enforce-mountable-secrets}
+
+例子：`kubernetes.io/enforce-mountable-secrets: "true"`
+
+用于：ServiceAccount
+
+此注解的值必须为 **true** 才能生效。此注解表示作为此服务帐户运行的 pod 只能引用在服务帐户的 `secrets` 字段中指定的 Secret API 对象。
+
+### controller.kubernetes.io/pod-deletion-cost {#pod-deletion-cost}
+
+例子：`controller.kubernetes.io/pod-deletion-cost=10`
+
+用于：Pod
+
+该注解用于设置 [Pod 删除成本](/docs/concepts/workloads/controllers/replicaset/#pod-deletion-cost) 允许用户影响 ReplicaSet 缩减顺序。 注解解析为 `int32` 类型。
+
+### beta.kubernetes.io/instance-type (已弃用)
+
+{{< note >}} 从 v1.17 开始，此标签已弃用，取而代之的是 [node.kubernetes.io/instance-type](#nodekubernetesioinstance-type)。 {{< /note >}}
+
+### node.kubernetes.io/instance-type {#nodekubernetesioinstance-type}
+
+例子：`node.kubernetes.io/instance-type=m3.medium`
+
+用于：Node
+
+Kubelet 使用 `cloudprovider` 定义的实例类型填充它。
+仅当你使用 `cloudprovider` 时才会设置此项。如果你希望将某些工作负载定位到某些实例类型，则此设置非常方便，但通常你希望依靠 Kubernetes 调度程序来执行基于资源的调度。你应该基于属性而不是实例类型来调度（例如：需要 GPU，而不是需要 `g2.2xlarge`）。
+
+### failure-domain.beta.kubernetes.io/region (已弃用) {#failure-domainbetakubernetesioregion}
+
+请参阅 [topology.kubernetes.io/region](#topologykubernetesioregion)。
+
+{{< note >}} 从 v1.17 开始，此标签已弃用，取而代之的是 [topology.kubernetes.io/region](#topologykubernetesioregion)。 {{</note>}}
+
+### failure-domain.beta.kubernetes.io/zone (已弃用) {#failure-domainbetakubernetesiozone}
+
+请参阅 [topology.kubernetes.io/zone](#topologykubernetesiozone)。
+
+{{< note >}} 从 v1.17 开始，此标签已弃用，取而代之的是 [topology.kubernetes.io/zone](#topologykubernetesiozone)。 {{</note>}}
+
+### statefulset.kubernetes.io/pod-name {#statefulsetkubernetesiopod-name}
+
+例子：`statefulset.kubernetes.io/pod-name=mystatefulset-7`
+
+当 StatefulSet 控制器为 StatefulSet 创建 Pod 时，控制平面会在该 Pod 上设置此标签。标签的值是正在创建的 Pod 的名称。
+
+有关详细信息，请参阅 StatefulSet 主题中的 [Pod 名称标签](/docs/concepts/workloads/controllers/statefulset/#pod-name-label)。
+
+### topology.kubernetes.io/region {#topologykubernetesioregion}
+
+例子：`topology.kubernetes.io/region=us-east-1`
+
+请参阅 [topology.kubernetes.io/zone](#topologykubernetesiozone)。
+
+### topology.kubernetes.io/zone {#topologykubernetesiozone}
+
+例子：`topology.kubernetes.io/zone=us-east-1c`
+
+用于：Node, PersistentVolume
+
+在 Node 上：`kubelet` 或外部`cloud-controller-manager` 使用 `cloudprovider` 提供的信息填充它。仅当你使用 `cloudprovider` 时才会设置此项。但是，如果它在你的拓扑中有意义，你应该考虑在 Node 上设置它。
+
+在 PersistentVolume 上：拓扑感知卷配置器将自动在 `PersistentVolumes` 上设置 Node 亲和性约束。
```

Comment:
I'd use singular terms for API kinds; for example:
```suggestion
在 PersistentVolume 上：拓扑感知卷配置器将自动在 `PersistentVolume` 上设置 Node 亲和性约束。
```

because English plurals aren't important to a document in Chinerse.

Reply:
Ok